burn-ndarray/src/ops/base.rs

```diff
@@ -392,38 +392,44 @@ where
         arg(tensor, dim, CmpType::Min)
     }
 
-    pub fn clamp_min<const D: usize>(tensor: NdArrayTensor<E, D>, min: E) -> NdArrayTensor<E, D> {
-        let array = tensor.array.mapv(|x| match x < min {
+    pub fn clamp_min<const D: usize>(
```

Comment:
This stuff was technically unrelated and could be removed - but I was experimenting with parallel maps in `ndarray`.